databases/orientdb/src/main/resources/config/orientdb-server-config.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <orient-server>
 	<handlers>
-		<handler class="com.orientechnologies.orient.server.hazelcast.OHazelcastPlugin">
+		<handler class="com.gentics.mesh.graphdb.cluster.MeshOHazelcastPlugin">
```

Comment:
Changelog entry? How to apply this on existing system? Automatic Changelog Job?

Reply:
@philippguertler Do you think a changelog entry (changelog.adoc) is enough to cover this? Mesh clustering will currently fail with a meaningful message when this change has not been applied. Or should we write a task that automatically migrates the config?

Reply:
Is it possible to start orientdb and ignore this entry in the config file? So that we can hard code the use of our plugin

Reply:
@philippguertler No - That is not possible.

Reply:
Then we have to automatically change the configuration file and fallback to fail with a meaningful message if the file is not writeable (e.g. readonly volume). Otherwise we would create a bigger breaking change.

Reply:
@philippguertler Okay. I have added the automatic changelog entry.